File: samples/FsBulletML.Sample.Unity2D.FSharp/FsBulletML.Sample.Unity2D.FSharp/Bomb.fs
namespace FsBulletML.Sample.Unity2D.FSharp

open System
open System.Collections.Generic
open System.Diagnostics 
open UnityEngine
open FsBulletML
open FsBulletML.Unity2D 

type Bomb =
  static member GenerateBomb(bombType:GameObject, position:Vector3) = 
    InstanceManager.InstantiatePrefab (bombType, position, Quaternion.identity) |> ignore

File: samples/FsBulletML.Sample.Unity2D.CSharp/Assets/Scripts/Bomb.cs
using UnityEngine;

public class Bomb : MonoBehaviour
{
    public static void GenerateBomb(GameObject BombType, Vector3 position)
    {
        InstanceManager.InstantiatePrefab(BombType, position, Quaternion.identity);
        AudioManager.PlaySE(0);
    }
}
